src/electron/main.tsx

```diff
@@ -16,6 +16,9 @@ import chiaConfig from '../util/config';
 import { i18n } from '../config/locales';
 import About from '../components/about/About';
 import packageJson from '../../package.json';
+import AppIcon from '../assets/img/chia64x64.png'
```

Comment:
Minor: semicolon at end?

```suggestion
import AppIcon from '../assets/img/chia64x64.png';
```

Reply:
Wow. My mistake. Fixed.